src/emulator/modules/SceDisplay/SceDisplay.cpp

```diff
@@ -94,15 +94,25 @@ EXPORT(int, sceDisplayWaitSetFrameBufCB) {
 }
 
 EXPORT(int, sceDisplayWaitSetFrameBufMulti) {
-    return UNIMPLEMENTED();
+    STUBBED("move after setframebuf");
+    if (!host.cfg.sync_rendering) {
```

Comment:
can't you do `return sceDisplayWaitSetFrameBuf();` here?

Reply:
yes :)

Reply:
ah after try i can't, it said definition not found
![image](https://user-images.githubusercontent.com/5261759/58887220-8516af80-86e5-11e9-936f-7ab246c022a0.png)